### ISSUE TITLE
PHP8 - Multi line attributes - Is the empty line afterwards by design?

### ISSUE THREAD

**Issue by AndreasA** [prettier/plugin-php#1884]
Using PHP8 with mutli line attributes result in formatting like this by prettier:

```php
    #[
        ORM\Column(...),
        ORM\CustomIdGenerator(class: ...),
        ORM\GeneratedValue(strategy: 'CUSTOM'),
        ORM\Id
    ]

    private string $id;
```

however, for single line attributes it is like this:

```php
    #[ORM\Column(...)]
    private string $id;
```

**Prettier 2.5.1**

**PHP Plugin 0.17.6**

`.prettierrc`
```json
{
    "$schema": "http://json.schemastore.org/prettierrc",
    "arrowParens": "always",
    "braceStyle": "psr-2",
    "bracketSameLine": false,
    "bracketSpacing": true,
    "phpVersion": "8.0",
    "printWidth": 120,
    "singleQuote": true,
    "tabWidth": 4,
    "trailingComma": "es5",
    "trailingCommaPHP": true
}

```

It is weird that the multi-line syntax adds an empty new line afterwards, but the single line does not. 

Is this by design or a bug?

This issue might be related: https://github.com/prettier/plugin-php/issues/1885

**Comment by czosel** [comment]
I think this is a bug, but I'd be curious to hear @cseufert's thoughts on this.

**Comment by AndreasA** [comment]
I just noticed that also the trailing comma is missing, which I think is a bug as well, so I created https://github.com/prettier/plugin-php/issues/1884

**Comment by AndreasA** [comment]
> I think this is a bug, but I'd be curious to hear @cseufert's thoughts on this.
> 
> Edit: [Playground example](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCACsAOj4gWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswbAgAWgAmJxBoAEdBeGgA12V-PTkFEGIRCBgytgq2KoMAXxYoCngYdBQ0eCxJUDZsGDaRTtrMMEEJvBAAHgB+YzAAHUwtmNkx7AACADF0dAPgLYODy6uDgGIAbRvb2+QAJQBZDY3vWUEfTBMZ4vA7vL4-YR9HwASQoAHEoJgkGw+hAABS7fZ4a5RNgQDYgACUQMwINen2+CKRrh0FFMcgiaPGNKgyiS2IA5J4AKrxUTID4c4nAl5g76w4EAXWBRAAbiioAdmSoDgASSgAbi27RA7SAA)

Thanks for the playground example. I think the language level is set wrong though (5.4), it should be correct [here](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCDoA6BiBYwIbeLJUBhdAFtvbAAoAEv74zoKGIBwuAMZQAMowAJ5q1mDYEAC0AEyOINAAjoLw0P4uyn56cgogxCIQMKVs5WyVBgC+LFAU8DDoKGjwWJKgbNgwrSIdNZhgguN4IAA8APzGYAA6mJvRsqPYAAQAYujo+8Cb+-sXl-sAxADa1zc3yABKALLr616ygt6YTCez32b0+32EvW8AEkKABxKCYJBsXoQAAUOz2eCukTYEHWIAAlIDMMCXh8vvDES4dBRTHJwqixtSoMpEliAOQeACqcVEyHe7KJQOeoK+MKBAF0gUQAG7IqD7JkqfYAEkoAG5Nm0QG0gA).

**Comment by cseufert** [comment]
@AndreasA Definitely a bug

**Comment by AndreasA** [comment]
@cseufert I think the Pr references the wrong bug? as it fixes the missing dangling comma: https://github.com/prettier/plugin-php/issues/1885

This issue is regarding the empty line between the property and the attribute closing bracket.

**Comment by czosel** [comment]
Fixed in `v0.18.0` :tada:
